nntrainer/src/databuffer.cpp

```diff
@@ -138,14 +217,20 @@ bool DataBuffer::getStatus(BufferType type) {
   case BUF_TRAIN:
     if ((train_data.size() < mini_batch) && trainReadyFlag)
       ret = false;
+    if (globalExceptionPtr)
```

Comment:
Can we hoist those overlapping `if (globalExceptionPtr)...` to line 215?

Reply:
right. Thanks.